flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaConsumerTestBase.java

```diff
@@ -2242,4 +2337,47 @@ public String getTargetTopic(Tuple3<Integer, Integer, String> element) {
 			return element.f2;
 		}
 	}
+
+	private static class TestDeSerializer implements KafkaDeserializationSchema<Tuple3<Integer, Integer, String>>,
```

Comment:
Minor:
Can we have a private static deserializer class (may be abstract). And let `TestDeserializer` and `Tuple2WithTopicSchema` extends that deserializer class but implement different serializer interface?

Reply:
done